sound/soc/sof/intel/hda-dai.c

```diff
@@ -442,9 +442,9 @@ static const struct snd_soc_dai_ops hda_link_dai_ops = {
 
 #endif
 
-/* the SSP dma_data is only using a flag for now */
 struct ssp_dai_dma_data {
-	bool suspended;
+	bool configured; /* used for hw_params/hw_free */
```

Comment:
@plbossart I already have a configured flag for snd_sof_dai that takes cares of this. 
https://github.com/thesofproject/linux/blob/52e8df0bc4e66bafd49d3ed9602e3c5d413b7539/sound/soc/sof/intel/hda.c#L110